src/rgw/driver/rados/rgw_notify.cc

```diff
@@ -1331,18 +1357,32 @@ int get_persistent_queue_stats(const DoutPrefixProvider *dpp, librados::IoCtx &r
 {
   // TODO: use optional_yield instead calling rados_ioctx.operate() synchronously
   cls_2pc_reservations reservations;
-  auto ret = cls_2pc_queue_list_reservations(rados_ioctx, queue_name, reservations);
+
+  auto io_context = boost::asio::io_context();
+  auto conn = rgw::redis::RGWRedis(io_context).get_conn();
+  // auto ret = cls_2pc_queue_list_reservations(rados_ioctx, queue_name, reservations);
+  // if (ret < 0) {
+  //   ldpp_dout(dpp, 1) << "ERROR: failed to read queue list reservation: " << ret << dendl;
+  //   return ret;
+  // }
+
+  std::tuple<int, int> reservation_status;
```

Comment:
could these stats be negative?
probably change from `int`

Reply:
ok